### PR TITLE
Repo: Updated URLs of sites that support HTTPS.

### PR DESCRIPTION
**Pull request by EricDunsworth**
This pretty much covers all of this repo's URLs, but excludes the following:
- Sites that lack HTTPS support
- Sites with broken HTTPS support (e.g. site looks broken when accessed via HTTPS)
- Sites that no longer exist
- Proxy URL patterns (which generally don't play nicely with https: prefixes)
- XMLNS attributes (http:-prefixed values are literally the names of the namespaces)
- vocab="http://schema.org/" attributes (was asked to stick with HTTP for them)

### CODE REVIEW

File: site/layouts/core.hbs [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		{{>metadata}}
 		{{>headresources}}
 	</head>
-	<body {{#if pageclass}}class="{{pageclass}}" {{/if}}vocab="http://schema.org/" typeof="WebPage">
+	<body {{#if pageclass}}class="{{pageclass}}" {{/if}}vocab="https://schema.org/" typeof="WebPage">
```
Comment (duboisp) [inline site/layouts/core.hbs]: The schema.org vocabulary should use "http://" scheme, not the "https://" 

reference: https://www.w3.org/2011/rdfa-context/rdfa-1.1

File: site/layouts/servermessage.hbs [hunk 2]
```diff
@@ -12,7 +12,7 @@
 		<meta name="robots" content="noindex, nofollow, noarchive" />
 		{{>headresources}}
 	</head>
-	<body vocab="http://schema.org/" typeof="WebPage">
+	<body vocab="https://schema.org/" typeof="WebPage">
```
Comment (duboisp) [inline site/layouts/servermessage.hbs]: The schema.org vocabulary should use "http://" scheme, not the "https://" 

reference: https://www.w3.org/2011/rdfa-context/rdfa-1.1

**Review by duboisp — CHANGES_REQUESTED** [review]

File: site/layouts/core.hbs (hunk 1 above)
Comment (EricDunsworth) [reply site/layouts/core.hbs]: I changed the vocab attribute's protocol was due to the [question/answer about HTTPS on schema.org's FAQ](https://schema.org/docs/faq.html#19).

**Excerpt:**
> **Should we write ``https://schema.org`` or ``http://schema.org`` in our markup?**
> 
> There is a general trend towards using ``https`` more widely, and you can already write ``https://schema.org`` in your structured data. Over time we will migrate the schema.org site itself towards using https: as the default version of the site and our preferred form in examples. However ``http://schema.org`` -based URLs in structured data markup will remain widely understood for the forseeable future and there should be no urgency about migrating existing data. This is a lengthy way of saying that both ``https://schema.org`` and ``http://schema.org`` are fine.

Thoughts?

File: site/layouts/servermessage.hbs (hunk 2 above)
Comment (EricDunsworth) [reply site/layouts/servermessage.hbs]: See my other conversation comment.

**Comment by EricDunsworth** [comment]
Just fixed a few issues I spotted:
- Reverted unrelated changes to package-lock.json
- Reverted an unrelated change in src/other/tablevalidator/demo/tablevalidator.js
- Reverted changes to src/polyfills/slider/assets/fd-slider-sprite.png (contains some XMLNS attributes I initially changed to HTTPS and backtracked on - but the file continued to be tracked as modified)
- Reverted some geogratis.gc.ca URLs back to HTTP (since they're tied to redirects that don't work via HTTPS)

**Comment by GormFrank** [comment]
Pre-approved during weekely technical review as long as the schema.org link is switched back to "http".

File: site/layouts/core.hbs (hunk 1 above)
Comment (EricDunsworth) [reply site/layouts/core.hbs]: Reverted this change as per our follow-up discussion.

File: site/layouts/servermessage.hbs (hunk 2 above)
Comment (EricDunsworth) [reply site/layouts/servermessage.hbs]: Reverted this change as per our follow-up discussion.

**Comment by EricDunsworth** [comment]
@duboisp @GormFrank FYI I've reverted the schema.org link and rebased to resolved merge conflicts. Should be good to merge once Travis-CI has finished rebuilding.

**Review by duboisp — APPROVED** [review]

**Pull request merged**